travis error

The travis build fails with this error:
```
make: *** No rule to make target `/closure/goog/transpile.js', needed by `.build/examples-hosted/lib/transpile.js'.  Stop.
```

It can be reproduce local with:
```
make cleanall
make examples-hosted-ngeo
```

When running `make examples-hosted-ngeo` the closure library is not yet installed so the `CLOSURE_LIBRARY_PATH` is empty

--- a/buildtools/deploy.sh
+++ b/buildtools/deploy.sh
@@ -1,5 +1,7 @@
 #!/bin/bash -ex
 
+make .build/node_modules.timestamp
+
 if [ "${EXAMPLES_NGEO}" == TRUE ]
 then
     make examples-hosted-ngeo .build/examples-hosted/index.html

